src/libraries/System.Private.CoreLib/src/System/Numerics/Vector.cs

```diff
@@ -88,11 +88,11 @@ public static Vector<T> Add<T>(Vector<T> left, Vector<T> right)
         public static Vector<T> AndNot<T>(Vector<T> left, Vector<T> right)
             where T : struct => left & ~right;
 
-        /// <summary>Reinterprets a <see cref="Vector64{T}" /> as a new <see cref="Vector64{U}" />.</summary>
+        /// <summary>Reinterprets a <see cref="Vector{T}" /> as a new <see cref="Vector{U}" />.</summary>
         /// <typeparam name="TFrom">The type of the input vector.</typeparam>
         /// <typeparam name="TTo">The type of the vector <paramref name="vector" /> should be reinterpreted as.</typeparam>
         /// <param name="vector">The vector to reinterpret.</param>
-        /// <returns><paramref name="vector" /> reinterpreted as a new <see cref="Vector64{U}" />.</returns>
+        /// <returns><paramref name="vector" /> reinterpreted as a new <see cref="Vector{U}" />.</returns>
```

Comment:
https://github.com/dotnet/runtime/pull/75032 is renaming the T and U to be TFrom and TTo in similar docs.  Presumably we should do so here as well.
cc: @gewarren

Reply:
I see that the type param name used in a `see cref` doesn't actually affect [how it's displayed in docs.microsoft.com](https://docs.microsoft.com/en-us/dotnet/api/system.numerics.vector.as?view=net-6.0) - so perhaps the only reason to update it is when reading the source code.